assets/admin/tour/lesson-tour/steps.js

```diff
@@ -149,7 +352,7 @@ export default function getTourSteps() {
 				),
 				descriptions: {
 					desktop: __(
-						'Click + to open the inserter. Then click the New Question or Existing Question(s) option.',
+						'Click + to open the inserter. Then click the New Question, Category Question(s) or Existing Question(s) option.',
```

Comment:
I added the "Category Question(s)" option here just to match all the options the user will see.

Reply:
It was actually left out intentionally because Category Questions are not used that much. Maybe leaving it out is okay? WDYT?

Reply:
Ah, okay! Reverting the change here: https://github.com/Automattic/sensei/pull/7538/commits/76f4a1b6c40f5654c9398e3adc5db57d2312eeee 😉